packages/core/src/tracing/trace.ts

```diff
@@ -33,7 +33,11 @@ interface StartSpanOptions extends TransactionContext {
   /** An op for the span. This is a categorization for spans. */
   op?: string;
 
-  /** The origin of the span - if it comes from auto instrumenation or manual instrumentation. */
+  /**
+   * The origin of the span - if it comes from auto instrumentation or manual instrumentation.
+   *
+   * @deprecated Set `attributes['sentry.origin']` instead.
```

Comment:
l: Should we reference the semantic attribute const here instead? Is probably a bit safer..?

Reply:
ah right, we export them. Sure, will do!